CorsixTH/Lua/languages/french.lua

```diff
@@ -16,19 +16,18 @@ FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
 AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
 LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
 OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
-SOFTWARE. --]]
-
-Language(utf8 "Français", "French" , "fr", "fre", "fra")
+SOFTWARE. --]] Language(utf8 "Français", "French", "fr", "fre", "fra")
```

Comment:
```suggestion
SOFTWARE. --]]

Language(utf8 "Français", "French", "fr", "fre", "fra")
```
I'd keep the spacing here